cde-to-odm/xsparql/cde-sex-to-odm.xsparql

```diff
@@ -0,0 +1,48 @@
+prefix obo: <http://purl.obolibrary.org/obo/>
+prefix sio: <http://semanticscience.org/resource/>
+prefix xsd: <http://www.w3.org/2001/XMLSchema#>
+prefix snomedct: <http://purl.bioontology.org/ontology/SNOMEDCT/>
+prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+
+<ClinicalData StudyOID="?" MetadataVersionOID="?">
+    <SubjectData SubjectKey="?" TransactionType="Insert">
+        <StudyEventData StudyEventOID="Screen">
+            <FormData FormOID="DEMOG">
+                <ItemGroupData>
+                    {
+                      for distinct $sex from <examples/cde-rdf/cde-dob-sex.ttl>
+                      where {
+                             {
+                               $person rdf:type sio:SIO_000498.
+                               $person sio:SIO_000217 $quality .
+
+                               $quality rdf:type obo:NCIT_C17357 .
+                               $quality sio:SIO_000642  $output.
+
+                               $output rdf:type sio:SIO_000340.
+                               $output rdf:type $sex.
+                             }
+                             union
```

Comment:
As there are two pathways to reach the "sex" quality in the CDE model, I choose UNION to represent both pathways. But it might be a next step to think about :)

Reply:
ah ok, but still you can do it with `UNION` right. Because according to CDE ShEx both these paths should exist in a valid CDE RDF.